contentcuration/contentcuration/api.py

```diff
@@ -7,9 +7,21 @@
 
 from django.core.files import File as DjFile
 from django.db.models import Q
+from django.http import HttpResponse
 from kolibri.content import models as KolibriContent
 from kolibri.content.utils import validate
 from kolibri.content.api import *
+from django.db import transaction
+import models
+
+def count_files(node):
```

Comment:
we my be able to leverage the fact that the node passed in here is a MPTTModel, so we can use its built in method `get_leafnodes` to get a queryset of all non-topic nodes, then when we count File objects, we can only hit the database once by doing something like `models.File.objects.filter(contentnode_in=<the_qureyset_mentioned_above>).count()